src/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ if (JSON_VALIDATOR_INSTALL)
                 NAMELINK_COMPONENT nlohmann_json_schema_validator_Development
                 ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT nlohmann_json_schema_validator_Development
                 PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/nlohmann COMPONENT nlohmann_json_schema_validator_Development
-                RUNTIME DESTINATION ${CMAKE_INSTALL_RUNTIMEDIR} COMPONENT nlohmann_json_schema_validator_Runtime)
+                RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT nlohmann_json_schema_validator_Runtime)
```

Comment:
Actually, can you get rid of `WIN32` and merge these two? If `LIBRARY DESTINATION` is not used on `WIN32` and `RUNTIME DESTINATION` is not defined for `UNIX` libraries, these two look identical right? Double-check if I am not dyslexic.

Reply:
Comparing the definitions with each other, the only difference I could find was the added `RUNTIME DESTINATION`.